rviz_default_plugins/src/rviz_default_plugins/displays/axes/axes_display.cpp

```diff
@@ -102,7 +102,7 @@ void AxesDisplay::update(float dt, float ros_dt)
   Ogre::Vector3 position;
   Ogre::Quaternion orientation;
   if (context_->getFrameManager()->getTransform(
-      frame, context_->getClock()->now(), position, orientation))
+      frame, position, orientation))
```

Comment:
nit (pre-existing) Documentation for this function seems insufficient:
https://github.com/ros2/rviz/blob/4e976dc444e7e59ec9cdef30f1f04e0beafcafc8/rviz_common/include/rviz_common/frame_manager_iface.hpp#L152-L165

Is it possible to update that docstring to mention it's effectively using `TimePointZero`?

Reply:
Updated docs in https://github.com/ros2/rviz/pull/893